core/chain-configs/src/genesis_config.rs

```diff
@@ -264,6 +266,37 @@ impl GenesisRecords {
     }
 }
 
+pub struct GenesisJsonHasher {
+    pub digest: sha2::Sha256,
```

Comment:
Do we actually need to make this field public?

Reply:
Yeah, better make it private